spec/rails_helper.rb

```diff
@@ -11,16 +11,16 @@
 
   def stub_omniauth
     OmniAuth.config.test_mode = true
-    OmniAuth.config.mock_auth[:google] = OmniAuth::AuthHash.new({
-      provider: “google”,
+    OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new({
+      provider: "google",
       uid: "12345678910",
       info: {
-        email: "homies@shelteredhomies.com”,
-        first_name: “Phillis”,
-        last_name: “Holland”
+        email: "homies@shelteredhomies.com",
+        first_name: "Phillis",
+        last_name: "Holland"
       },
       credentials: {
-        token: “abcdefg12345",
+        token: "abcdefg12345",
         refresh_token: "12345abcdefg",
```

Comment:
@erose357 @jtruong2 @swdonovan Do we need to include the `refresh_token` with our stub user? Didn't we remove this from the User model?